core/common/libp2p/cbor_stream.cpp

```diff
@@ -24,7 +24,9 @@ namespace fc::common::libp2p {
                             WriteCallbackFunc cb) {
     stream_->write(*input,
                    input->size(),
-                   [input, cb{std::move(cb)}](auto written) { cb(written); });
+                   [input{std::move(input)}, cb{std::move(cb)}](auto written) {
```

Comment:
is there use-after-move in `*input` argument?

Reply:
fixed